Bookish/Views/Book/_BookTable.cshtml

```diff
@@ -4,27 +4,34 @@
     <table class="table">
         <thead>
         <tr>
-            <th scope="col">Id</th>
-            <th scope="col">Author</th>
-            <th scope="col">Title</th>
+            @foreach (var property in typeof(BookModel).GetProperties())
+            {
+                <th scope="col">@property.Name</th>
+            }
         </tr>
         </thead>
         <tbody>
         @foreach (var book in Model)
         {
             <tr>
                 <th scope="row">@book.Id</th>
-                <td>@book.Author</td>
-                <td>@book.Title</td>
+                @foreach (var property in typeof(BookModel).GetProperties())
+                {
+                    if (property.Name != "Id")
+                    {
+                        <td>@property.GetValue(book)</td>
+                    }
+                }
+                
                 <td>
-                    @using (Html.BeginForm("EditBook", "Home", FormMethod.Get))
+                    @using (Html.BeginForm("EditBook", "Book", FormMethod.Get))
                     {
                         <input type="hidden" id="id" name="id" value=@book.Id>
                         <button type="submit" class="btn btn-primary">Edit</button>
                     }
                 </td>
                 <td>
-                    @using (Html.BeginForm("DeleteBook", "Home", FormMethod.Post))
+                    @using (Html.BeginForm("DeleteBook", "Book", FormMethod.Post))

```

Comment:
It looks like this button immediately deletes a book. A nice extension would be for it to go to a confirmation page

Reply:
It's in the list of things we plan on doing when there's time